README.md

```diff
@@ -505,12 +425,103 @@ await asyncFind(animal => animal.kind === 'dog', [
 ]) // {type: 'dog'}
 ```
 
+## tap
+Tap is not unlike a forEach method, and like forEach is usually used to express side effects. Without breaking a chain of composition, it allows you access to the value yielded to it. Tap always yields the same value it received. Tap can be curried.
+```js
+compose(
+  tap(item => console.log(item)),
+  filter(item => !!item),
+)([0, 1, 2]) // logs "1", "2". returns Iterable[1, 2]
+```
+
+## async-tap
+Same as tap, but for async iterables
+```js
+compose(
+  asyncTap(item => console.log(item)),
+  asyncFilter(item => !!item),
+)(asyncIter([0, 1, 2])) // logs "1", "2". returns AsyncIterable[1, 2]
+```
+
 ## size
 Returns the number of values yielded by an iterable. Without loading the whole sequence in memory. It works for both sync and async iterables.
```

Comment:
is it true that works for async-iterable? why async-size exists, then ?

Reply:
Oops, no, of course not.